flake8_length/_checker.py

```diff
@@ -1,38 +1,46 @@
 # built-in
 import tokenize
-from typing import Iterator, NamedTuple, Sequence, Tuple
+from typing import Iterator, NamedTuple, Sequence, Tuple, Literal
 
 # app
 from ._parser import get_lines_info
 
 
 Tokens = Sequence[tokenize.TokenInfo]
-TEMPLATE = 'LN001 line is too long ({v.length} > {v.limit})'
+VIOLATION_DESCRIPTION_MAP = {
+    'LN001': 'code line is too long',
+    'LN002': 'doc/comment line is too long'
+}
+
+TEMPLATE = '{v.code} {d} ({v.length} > {v.limit})'
 
 
 class Violation(NamedTuple):
+    code: Literal['LN001', 'LN002']
```

Comment:
`Literal` is new in 3.8, the current stable is 3.7, let's just use `str` there.

```suggestion
    code: str
```